force_estimator:
  rate: 100.0
  use_momentum_based: false
  is_model_floating_base: true
  model_type: RBDL
  links: [arm1_6, arm2_6]
  chains: [left_arm, right_arm]
  arm1_6:
    dofs: [0, 1, 2]
  arm2_6:
    dofs: [0, 1, 2]
  ignored_joints: []
  svd_threshold: 0.05
  #reset offset
  sec_to_reset: 3
  #log
  enable_log: false
  #ref frame wrt forces are published
  ref_frame: pelvis
  #filter
  enable_filter: true
  filter_damp: 0.8
  filter_bw: 3
  filter_dead_zone: 0
  #markers
  pub_arrows: true
  #The scale factor will scale the incoming forces vector to show the arrow with a scaled magnitude.
  #Then, after the scaling, if the arrow has a norm > arrow_max_norm, the arrow norm will be saturated.
  #Consider that arrow_max_norm are meters, so set these values according to robot size for a useful visualization
  arrow_scale_factor: 20  # The number will DIVIDE each component, so bigger == shorter arrows
  arrow_max_norm: 3
  